docs/USAGE-GUIDE.md

```diff
@@ -486,6 +486,30 @@ To use certificate transparency, the requester (usually the client) must call `s
 
 Call `s2n_connection_get_sct_list()` to retrieve the received certificate transparency information. The format of this data is the SignedCertificateTimestampList structure defined in section 3.3 of RFC 6962.
 
+## Session Resumption
+
```

Comment:
We should probably add a section on the session ticket key. How does it rotate? Any rules for generating it?

Also, can you describe how a client retrieves the tickets? You mostly talk about how to set them.

Reply:
Okay I added this section, but rules for generating it? Do we have those rules?